BlockEntity/BEFruitpress.cs

```diff
@@ -425,6 +427,16 @@ private bool InteractMashContainer(IPlayer byPlayer, BlockSelection blockSel)
             // Take out mash
             if (MashSlot.Empty) return false;
 
+            if (juiceableLitresLeft < 0.01 && mashStack.Collectible.Code.Path != "rot")

```

Comment:
I moved this block of code to when the player removes the mash so you never end up in the situation where the player is trying to add more fruit while there is a stack of finished dry mash in the press. It also removes squeezeRel fixing the stacking bug.